RoboJackets/robocup-software
# Add Code section to Contributions Doc

## Description
I transferred and editing the old contributing guide in robocup-software/doc. It has relevant, concise information with less confusing images. [Here](https://rj-rc-software.readthedocs.io/en/contributing_guidelines/) is the link to the docs.

## Steps to test
### Test Case 1
1. Open link and check for typos.
2. Make sure content makes sense
3. Let me know if there is a section or pictures that I should add
